Gems/Atom/Tools/PassCanvas/Code/Source/PassCanvasApplication.cpp

```diff
@@ -107,7 +107,7 @@ namespace PassCanvas
     void PassCanvasApplication::Destroy()
     {
         // Save all of the graph view configuration settings to the settings registry.
-        AtomToolsFramework::SetSettingsObject("/O3DE/Atom/PassCanvas/GraphViewSettings", m_graphViewSettingsPtr);
+        AtomToolsFramework::SetSettingsObject("/O3DE/Atom/GraphView/ViewSettings", m_graphViewSettingsPtr);
```

Comment:
Why save over the GraphView settings, shouldn't the PassCanvas settings be independent of the MaterialCanvas settings and other tools?